administrator/language/de-DE/de-DE.com_users.sys.ini

```diff
@@ -4,9 +4,29 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt, see LICENSE.php
 ; Note : All ini files need to be saved as UTF-8
 
-COM_USERS_CONTENT_TYPE_USER="Benutzer"
-COM_USERS_CONTENT_TYPE_NOTE="Benutzer Notiz"
+COM_USERS="Benutzer"
 COM_USERS_CONTENT_TYPE_CATEGORY="Benutzer Notiz Kategorie"
+COM_USERS_CONTENT_TYPE_NOTE="Benutzer Notiz"
+COM_USERS_CONTENT_TYPE_USER="Benutzer"
+COM_USERS_GROUPS_VIEW_DEFAULT_DESC="Zeigt eine Liste von Benutzergruppen"
+COM_USERS_GROUPS_VIEW_DEFAULT_TITLE="Benutzergruppen"
+COM_USERS_GROUP_VIEW_EDIT_DESC="Ein Formular zum erstellen und bearbeiten von Benutzergruppen"
+COM_USERS_GROUP_VIEW_EDIT_TITLE="Benutzergruppen erstellen / bearbeiten"
+COM_USERS_LEVELS_VIEW_DEFAULT_DESC="Zeigt eine Liste von Zugriffsebenen"
+COM_USERS_LEVELS_VIEW_DEFAULT_TITLE="Zugriffsebenen"
+COM_USERS_LEVEL_VIEW_EDIT_DESC="Ein Formular zum erstellen und bearbeiten von Zugriffsebenen"
+COM_USERS_LEVEL_VIEW_EDIT_TITLE="Zugriffsebenen erstellen / bearbeiten"
+COM_USERS_MAIL_VIEW_DEFAULT_DESC="Ein Formular zum senden von Massenmails an Benutzer"
```

Comment:
ist aktuell in der Core übersetzung so issue: https://github.com/joomlagerman/joomla/issues/198